app/helpers/application_helper.rb

```diff
@@ -124,6 +124,13 @@ def make_line(row, td_opts)
   #
   def reload_with_args(new_args)
     uri = request.url.sub(/^\w+:\/+[^\/]+/, "")
+
+    # In the infnite scroll javascript, we want to make sure that the pagination html
+    # which is returned does not lead to the ajax controller.
+    if request.url.include? "ajax"
```

Comment:
These 3 lines can be shortened to 
`uri = request.url.sub(/ajax/, "observer") if request.url.include? "ajax"`